### PR TITLE
[DOCS] Add #135663 to 8.3.2 release notes

### PR DESCRIPTION
**Pull request by nastasha-solomon**
Addresses https://github.com/elastic/security-docs/issues/2188.

Preview [here](https://security-docs_2189.docs-preview.app.elstc.co/guide/en/security/master/release-notes-header-8.3.0.html#release-notes-8.3.2).

### CODE REVIEW

File: docs/release-notes/8.3.asciidoc [hunk 1]
```diff
@@ -10,6 +10,20 @@
 ==== Bug fixes and enhancements
 * Allows indices created from value lists to be used with indicator match rules ({pull}135128[#135128]).
 * Fixes a bug that prevented the *Create field* button from appearing in the Fields menu when you accessed it from a Timeline created using the Alerts page's *Open in timeline* button ({pull}135842[#135842]).
+* Fixes an issue where detection rules that were created or edited in 8.2.x failed execution after upgrading to {stack} to 8.3.0 or 8.3.1 ({pull}135663[#135663]).
+
+====
+
+[IMPORTANT]
+
+If you already upgraded to 8.3.0 or 8.3.1 and noticed that rules created or updated in 8.2.x were failing with an error similar to the message below, complete the appropriate steps for restoring your rules after you upgrade to 8.3.2. Refer to the <<known-issue-8.3.1, known issue section>> of the 8.3.1 release notes for more information.
```
Comment (nastasha-solomon) [inline docs/release-notes/8.3.asciidoc:19]:
```suggestion
If you already upgraded to 8.3.0 or 8.3.1 and noticed that rules created or updated in 8.2.x were failing with an error similar to the message below, complete the appropriate steps to restore your rules after you upgrade to 8.3.2. Refer to the <<known-issue-8.3.1, known issue section>> of the 8.3.1 release notes for more information.
```

File: docs/release-notes/8.3.asciidoc [hunk 2]
```diff
@@ -10,6 +10,20 @@
 ==== Bug fixes and enhancements
 * Allows indices created from value lists to be used with indicator match rules ({pull}135128[#135128]).
 * Fixes a bug that prevented the *Create field* button from appearing in the Fields menu when you accessed it from a Timeline created using the Alerts page's *Open in timeline* button ({pull}135842[#135842]).
+* Fixes an issue where detection rules that were created or edited in 8.2.x failed execution after upgrading to {stack} to 8.3.0 or 8.3.1 ({pull}135663[#135663]).
```
Comment (benironside) [inline docs/release-notes/8.3.asciidoc:13]:
```suggestion
* Fixes an issue where detection rules that were created or edited in 8.2.x failed to execute after you upgraded to {stack} to 8.3.0 or 8.3.1 ({pull}135663[#135663]).
```

**Review by benironside — COMMENTED** [review]
Looks good to me, left one suggestion.

File: docs/release-notes/8.3.asciidoc [hunk 3]
```diff
@@ -10,6 +10,20 @@
 ==== Bug fixes and enhancements
 * Allows indices created from value lists to be used with indicator match rules ({pull}135128[#135128]).
 * Fixes a bug that prevented the *Create field* button from appearing in the Fields menu when you accessed it from a Timeline created using the Alerts page's *Open in timeline* button ({pull}135842[#135842]).
+* Fixes an issue where detection rules that were created or edited in 8.2.x failed to execute after you upgraded to {stack} to 8.3.0 or 8.3.1 ({pull}135663[#135663]).
```
Comment (nastasha-solomon) [inline docs/release-notes/8.3.asciidoc:13]:
```suggestion
* Fixes an issue where detection rules that were created or edited in 8.2.x failed to execute after you upgraded to {stack} 8.3.0 or 8.3.1 ({pull}135663[#135663]).
```

**Review by jethr0null — APPROVED** [review]
LGTM!

**Review by peluja1012 — APPROVED** [review]
Looks good. Thank you!

**Pull request merged**